steps/cadence-innovus-flowsetup/setup.tcl

```diff
@@ -464,6 +464,10 @@ set vars(enable_cppr)   both
 
 set vars(signoff,verify_metal_density,skip) true
 
+# Skip fill when running verifyConnectivity
+
+set vars(signoff,verify_connectivity,replace_tcl)      $vars(custom_scripts_dir)/verify-connectivity.tcl
```

Comment:
Can you move this up to around line 380, where all the other custom scripts are?